src/gui/generalsettings.cpp

```diff
@@ -45,6 +45,14 @@
 #include <QOperatingSystemVersion>
 #endif
 
+#ifdef Q_OS_WIN
+#include "vfs_windows.h"
+#endif
+
+#ifdef Q_OS_MAC
+#include "vfs_mac.h"
+#endif
+
```

Comment:
Same here regarding the general vfs header idea (include "vfs.h" -> #ifdefs in vfs.h)